app/api/v1/endpoints/forecasts/router.py

```diff
@@ -57,6 +56,23 @@ def get_file_from_bucket(bucket_name : str, file_name:str):
         return blob.download_as_bytes()
     except Exception as e:
         return None
+    
+def upload_file_to_bucket(bucket_name : str,file: UploadFile, new_filename: str):
+    bucket = client.bucket(bucket_name)
+    blob = bucket.blob(new_filename)    
+    try:
+        blob.upload_from_file(file.file, content_type="application/pdf")
+        return {"status": "success", "message": f"PDF '{new_filename}' uploaded successfully"}
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error uploading PDF: {str(e)}")
```

Comment:
Reliance on Google Storage is something we should be careful with. Should keep the design open to different methods of storage. Supporting local file storage would help in development, and in environments where Google is not an option.

Consider moving the try/catch to the caller i.e. `upload_forecast` to have better separation of concerns. This function knows too much about how it is being used i.e. in a HTTP API.